tests/integration/test_author_name.py

```diff
@@ -0,0 +1,97 @@
+import pytest
+from splinter import Browser
+
+
+class TestAuthorName:
+    host = 'http://localhost:8080/works'
+
+    @pytest.fixture()
+    def browser(self):
+        browser = Browser('chrome')
+        yield browser
+        browser.quit()
+
+    def test_author_name_with_dates(self, browser):
+        url = self.host + '/OL118420W/Flatland'
+        browser.visit(url)
+        title = browser.find_by_css(".edition-byline")[0].text
+        begin_of_date = title.find('(')
+        if begin_of_date:
+            begin_of_date += 1
+        end_of_date = title.find(')')
+        date = title[begin_of_date:end_of_date]
+        assert date
+
+    def test_author_birth_and_death_year(self, browser):
+        url = self.host + '/OL118420W/Flatland'
+        browser.visit(url)
+        title = browser.find_by_css(".edition-byline")[0].text
+        begin_of_date = title.find('(')
+        if begin_of_date:
+            begin_of_date += 1
+        end_of_date = title.find(')')
+        date = title[begin_of_date:end_of_date]
```

Comment:
It would be cool to create a separate function from these lines...
```
def get_date_from_string(string):
    <this code here>
    return string[begin_of_date:end_of_date]
```